config.yaml

```diff
@@ -5,6 +5,36 @@ options:
   auto-delete:
     type: boolean
     description: |
-        When a relation is removed, auto-delete ensures that any relevant databases
-        associated with the relation are also removed
+      When a relation is removed, auto-delete ensures that any relevant databases
+      associated with the relation are also removed
     default: false
+
```

Comment:
I think it shouldn't change `config.yaml` here

Reply:
thanks for spotting this was an articifact from `git checkout -b`